packages/dhis2/src/Adaptor.js

```diff
@@ -36,10 +34,16 @@ export function execute(...operations) {
 
   return state => {
     const version = state.configuration?.apiVersion;
-    if (+version >= 42)
+
+    if (+version < 42) {
+      console.warn(
+        `WARNING: This adaptor uses the new tracker APIs for create, update, upsert and destroy. Options and parameters may change, see https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/tracker.html`
```

Comment:
@mtuchi I asked for this stuff but I find myself questioning its value this morning. Do you think we  should bother with this warning?

My consideration is: if you've got an existing workflow which works for `create('events')`, and then you update the adaptor version, the options you pass into `create()` may no longer work. Because we'll use the new tracker API behind-the-scenes.

This is a serious worry - but maybe it's for the changelog and not the console?

Reply:
Sorry I've answered my own question there :joy: I'll deal with it